applications/nrf5340_audio/src/modules/display/button_tab.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+enum button_pin_names {
```

Comment:
Try to include these from `button_assignments.h` instead to avoid multiple decalrations